src/site/markdown/server/manage-webadmin.md

```diff
@@ -2481,7 +2481,7 @@ Will return the full JSON associated with this event.
 Response codes:
 
  - 200: Success. A JSON representing this event is returned.
- - 400: Invalid group name or eventId
```

Comment:
`Will return a list of event ids:` above is wrong. You should state: `Will return a list of insertionIds`